src/main/scala/evaluation/AugmentedExamplesEvaluator.scala

```diff
@@ -0,0 +1,69 @@
+package evaluation
+
+import scala.reflect.ClassTag
+import breeze.linalg._
+import breeze.numerics._
+
+import org.apache.spark.rdd.RDD
+import nodes.util.MaxClassifier
+
+object AugmentedExamplesEvaluator extends Serializable {
```

Comment:
Instead of `AugmentedExamplesEvaluator` this seems like it could just as easily be called `EnsembleEvaluator` or something. 

In the traditional ensemble setting, a hypothetical `EnsembleEstimator` could be an `Estimator[T,Seq[V]]` or if people wanted to construct ensembles themselves, they could use a `Pipeline.gather` to coallate the results of running several models on an input data item.

Here we have a case that's slightly different, but I think the basic idea still holds.